Azure/sonic-buildimage
# [Accton] Fixes to AS4630-54TE sfp.py for sfputil usage

#### Why I did it
Fix the exception or malfunction while:
1. $ sfputil show eeprom
2. $ sfputil lpmode on/off EthernetXX
3. $ sfputil reset EthernetXX

#### How I did it
Referenced to AS4630-54PE's code in https://github.com/Azure/sonic-buildimage/pull/8241 majorly because they share use the same CPLD FW.

#### How to verify it
Confirm sfputil eeprom/lpmode/reset cmd can work.
At the same time, check if the CPLD reg values are the same as expected.
```
root@sonic:/home/admin# sfputil show eeprom
Ethernet0: SFP EEPROM not detected
Ethernet1: SFP EEPROM not detected
Ethernet2: SFP EEPROM not detected
...
Ethernet52: SFP EEPROM detected
        Application Advertisement: N/A
        Connector: Unknown or unspecified
        Encoding: Unspecified
        Extended Identifier: Power Class 1(1.5W max)
        Extended RateSelect Compliance: QSFP+ Rate Select Version 1
        Identifier: Unknown or unspecified
        Length Cable Assembly(m): 0
        Nominal Bit Rate(100Mbs): 0
        Specification compliance:
        Vendor Date Code(YYYY-MM-DD Lot): 20-- 
        Vendor Name: 
        Vendor OUI: 00-00-00
        Vendor PN: 
        Vendor Rev: 
        Vendor SN: 

Ethernet56: SFP EEPROM detected
        Application Advertisement: N/A
        Connector: No separable connector
        Encoding: Unspecified
        Extended Identifier: Power Class 4(3.5W max)
        Extended RateSelect Compliance: QSFP+ Rate Select Version 1
        Identifier: QSFP+ or later
        Length Cable Assembly(m): 0
        Nominal Bit Rate(100Mbs): 103
        Specification compliance:
                10/40G Ethernet Compliance Code: 40GBASE-CR4
        Vendor Date Code(YYYY-MM-DD Lot): 2014-11-06 
        Vendor Name: Accton
        Vendor OUI: cc-4e-24
        Vendor PN: F0OZZGIGA004
        Vendor Rev: 0B
        Vendor SN: CRH0000000000000

root@sonic:/home/admin# sfputil show lpmode -p Ethernet52
Port        Low-power Mode
----------  ----------------
Ethernet52  Off
root@sonic:/home/admin# sfputil lpmode on Ethernet52
Enabling low-power mode for port Ethernet52 ... OK
root@sonic:/home/admin# sfputil show lpmode -p Ethernet52
Port        Low-power Mode
----------  ----------------
Ethernet52  On
root@sonic:/home/admin# sfputil lpmode off Ethernet52
Disabling low-power mode for port Ethernet52 ... OK
root@sonic:/home/admin# sfputil show lpmode -p Ethernet52
Port        Low-power Mode
----------  ----------------
Ethernet52  Off

root@sonic:/# sfputil reset Ethernet56
Resetting port Ethernet56 ... OK

```

#### Which release branch to backport (provide reason below if selected)



- [x] 202012 : some of our customers demands it.
- [x] 202106 : some of our customers demands it.

#### Description for the changelog

[Accton] Fixes to AS4630-54TE sfp.py for sfputil usage

#### A picture of a cute animal (not mandatory but encouraged)
![Alt Text](https://media.giphy.com/media/ej4lfadivQe1W/giphy.gif)
